app/Console/Services/Importer.php

```diff
@@ -405,15 +418,41 @@ protected function importPireps()
 
     protected function importUsers()
     {
-        /*$this->comment('--- USER IMPORT ---');
+        $this->comment('--- USER IMPORT ---');
 
         $count = 0;
         foreach ($this->readRows('pilots') as $row)
         {
+            # TODO: What to do about pilot ids
+
+            $name = $row->firstname.' '.$row->lastname;
+            $airline_id = $this->airlines[$row->code];
+            $rank_id = $this->ranks[$row->rank];
```

Comment:
This gets the rank before it's imported, so the id remains as rankid in the old DB. Any ideas on how to change this to get the imported rank in the new DB?

Reply:
Ah good point! Let me think about this

Reply:
D'oh, I remember what I was meaning to do. I was gonna set the value to the new ID from the insert. I'll fix it later.

Reply:
All good, this might be the same with airlines as well (only tested with the one that was already there, so not sure).

Reply:
Yeah, I'll fix it. Good to merge?

Reply:
I think so, I'll add the admin permissions thingo in another branch after so it doesn't stuff the current stuff up.

Reply:
And also for that, importUsers function or separate function?

Reply:
Umm I gotta thing about that. I might have to do a bit of refactoring for the ID mapping stuff. Probably makes sense to go into the method that's resetting the password/api key, etc. But that too will likely be once I refactor the mapping thing in a couple of hours

Reply:
np, I'll leave it for now.